stage1/init/mount.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2014 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// file with functions that allow for creating a mount units for
+// managing inner and external volumes in stage1
+// mount units from systemd require to have /usr/bin/mount
+// TODO: it's would better to just use host@.mount empty@.mount systemd templates, but
+//		 technique requires 222 systemd
+// TODO: move path creation to path.go
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/coreos/go-systemd/unit"
+	"github.com/coreos/rkt/common"
+)
+
+const (
+	stage1MntDir = "/mnt/" // in this place on stage1 rootfs shared volumes will be put (or empty directories for kind=empty)
+)
+
+// newMountUnit utility function that creates new mount
+// unit in default systemd location (/usr/lib/systemd/system)
+// root is an relative path pod stage1 filesystem e.g. uuid/rootfs/ Pod.Root
+// requiredBy in install section creates required in given unit
+func newMountUnit(root, what, where, fsType, options, requiredBy string) error {
+
+	opts := []*unit.UnitOption{
+		unit.NewUnitOption("Unit", "Description", fmt.Sprintf("Mount unit for %s", where)),
+		unit.NewUnitOption("Unit", "DefaultDependencies", "false"),
+		unit.NewUnitOption("Mount", "What", what),
+		unit.NewUnitOption("Mount", "Where", where),
+		unit.NewUnitOption("Mount", "Type", fsType),
+		unit.NewUnitOption("Mount", "Options", options),
+	}
+
+	if requiredBy != "" {
+		opts = append(opts, unit.NewUnitOption("Install", "RequiredBy", requiredBy))
+	}
+
+	unitsPath := filepath.Join(root, "/usr/lib/systemd/system")
```

Comment:
Please use `unitsDir` from path.go instead of hardcoding `/usr/lib/systemd/system`.